.drone/drone.jsonnet

```
@@ -339,7 +339,7 @@ local manifest(apps) = pipeline('manifest') {
       run('LogQL', ['go test -mod=vendor -bench=Benchmark -benchtime 20x -timeout 120m ./pkg/logql/'])
     ],
     trigger: {event: ['cron'], cron: ['loki-bench']},
```

@julienduchesne there's something wrong here. It seems the job is not executed. Should this rather be
```suggestion
    trigger+: {
      event+: {
        include+: ['cron'],
      },
      cron+: {
        include+: ['loki-bench'],
      },
    },
```